packages/data-driver-postgres/src/index.ts

```diff
@@ -33,9 +33,9 @@ export default class DataDriverPostgres implements DataDriver {
 
 	async query(query: AbstractQuery): Promise<Readable> {
 		try {
-			const sqlStatement = convertAbstractQueryToSqlStatement(query);
-			const sqlString = constructSql(sqlStatement);
-			const queryStream = new QueryStream(sqlString /*, sqlStatement.parameters */);
+			const abstractSQL = convertAbstractQueryToSqlStatement(query);
+			const sqlQuery = constructSql(abstractSQL);
+			const queryStream = new QueryStream(sqlQuery.statement, sqlQuery.parameters);
```

Comment:
The naming of those variables seems to be a little off. Can we name them a little more accurately, maybe something like this?
```suggestion
			const genericSql = convertAbstractQueryToSqlStatement(query);
			const sql = constructSql(genericSql);
			const queryStream = new QueryStream(sql.statement, sql.parameters);
```

Reply:
I think so far we called it abstract SQL, so I would actually leave it as it is for now and not introduce the term generic now 🤔 
But the naming is indeed a bit off. I would like to highlight that a _query_ (`SELECT x FROM Y..`) is build here and not some other sort of statement (like one to insert or update, which will be added sooner or later). I just puhed some tweaks now 👀

Reply:
LGTM 👌